modules/restarting-installation.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// installing/installing-troubleshooting.adoc
+
+[id="restarting-installation_{context}"]
+= Reinstalling the {product-title} cluster
+
+If you are unable to debug and resolve issues in the failed {product-title} installation, you might need to consider installing a new {product-title} cluster. Before starting the installation process again, you must do a thorough cleanup.
+
+.Procedure
+
+1. Destroy the cluster and remove all the resources associated with the cluster, including the hidden installer state files in the installation directory:
++
+[source, terminal]
+----
+$ ./openshift-install destroy cluster --dir=<installation_directory> <1>
+----
+<1> `installation_directory` is the directory you specified when you ran `./openshift-install create cluster`. This directory contains the {product-title}
+definition files that the installation program creates.
```

Comment:
After running above comment, it's better to add this part "And if you want to reinstall from scratch, rm -rf the asset directory beforehand." suggested in dev doc https://github.com/openshift/installer/tree/ee89123890d894201145d0c5176a631885f08220#cleanup, especially for fresh UPI installation